web/frontend/src/app/component/hierarchy-display/hierarchy-display.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { HierarchyDisplayComponent } from './hierarchy-display.component';
 
-// Testing for HierarchyDisplayComponent (default tests)
+// Testing for HierarchyPopupComponent (default tests)
 describe('HierarchyDisplayComponent', () => {
```

Comment:
This comment should still say HierarchyDisplayComponent

Reply:
Fixed